src/System.Linq.Parallel/tests/QueryOperators/JoinTests.cs

```diff
@@ -255,7 +255,7 @@ public static void Join_Multiple_LeftWithOrderingColisions_UnorderedRight(Labele
 
         [Theory]
         [OuterLoop]
-        [MemberData(nameof(JoinOrderedLeftUnorderedRightData), new[] { 512, 1024 })]
+        [MemberData(nameof(JoinOrderedLeftUnorderedRightData), new[] { 256, 512 })]
```

Comment:
Are the original numbers significant in the implementation in some way? Do they cause some path to be tested that isn't otherwise tested, or cause some concurrency to happen that wouldn't otherwise?  Does this affect code coverage at all?

If there are no such adverse effects, LGTM. Thanks.

Reply:
I don't think changing this numbers will affect the code paths we are covering. I believe the original numbers was mainly meant as kind of stress scenario more than just testing the scenario ensuring we can handle such big lists in joining operations.